esbuild.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+require('esbuild')
+    .build({
+        bundle: true,
+        color: true,
+        entryPoints: ['src/main/index.js'],
+        external: ['electron', './node_modules/*'],
```

Comment:
Won't electron be in node_modules?

Reply:
Even though `electron` is in `node_modules` it must currently be included explicitly. I think this is necessary because `esbuild` currently rewrites module imports into relative paths. They want to change this eventually (evanw/esbuild#1958, evanw/esbuild#2164, evanw/esbuild#2246) and then we can probably remove it here.

Right now: If you remove it, build `main` and then try to run it you get an error about undefined properties.